TriblerGUI/widgets/downloadspage.py

```diff
@@ -431,13 +473,16 @@ def on_right_click_item(self, pos):
             self.selected_items.append(item_clicked)
 
         menu = TriblerActionMenu(self)
+        menu.setMinimumWidth(CONTEXT_MENU_WIDTH)
 
         start_action = QAction('Start', self)
         stop_action = QAction('Stop', self)
         remove_download_action = QAction('Remove download', self)
+        add_to_channel_action = QAction('Add to My Channel', self)
```

Comment:
Minor comment: we should be consistent in the spelling of "my channel". I think it's better to not capitalise the `M` and `C` since it may look like some piece of technology that we named.

Reply:
sure, done